packages/orchestrator-ui-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@orchestrator-ui/orchestrator-ui-components",
-    "version": "1.6.1",
+    "version": "1.7.0",
```

Comment:
Not sure if you should adjust package version manually. The changeset bot should do this